dxr/plugins/python/indexers.py

```diff
@@ -153,30 +227,35 @@ def __init__(self, path, contents, plugin_name, tree, python_path,
         self.derived_classes = derived_classes
         self.names = names
 
+        self.module_name = path_to_module(self.python_path, self.path)
+
     def is_interesting(self):
         return is_interesting(self.path)
 
     def needles_by_line(self):
+        self.analyze_tokens()
+
+        visitor = IndexingNodeVisitor(self)
+        syntax_tree = ast.parse(self.contents.encode('utf-8'))
```

Comment:
OTOH, I just rediscovered my comment below about inconsistencies, which frightens me more than any alleged reason of Jeff's, so yeah: let's be consistent.